### PR TITLE
Update source build version for Valhalla to JDK22

### PR DESCRIPTION
**Pull request by a7ehuo**
Update source build version for Valhalla to JDK22

### CODE REVIEW

**Comment by a7ehuo** [comment]
This PR should resolve the nightly [Valhalla test build failure ](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdkValhalla_j9_sanity.functional_x86-64_linux_valhalla_Nightly_testList_0/827/console) with `openj9-openjdk-jdk.valuetypes`. 

```
22:06:02      [javac] /home/jenkins/workspace/Test_openjdkValhalla_j9_sanity.functional_x86-64_linux_valhalla_Nightly_testList_0/aqa-tests/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeArrayTests.java:45: error: value classes are not supported in -source 20
22:06:02      [javac] 	static value class PointV implements SomeIface {
22:06:02      [javac] 	       ^
22:06:02      [javac]   (use -source 21 or higher to enable value classes)
22:06:02      [javac] /home/jenkins/workspace/Test_openjdkValhalla_j9_sanity.functional_x86-64_linux_valhalla_Nightly_testList_0/aqa-tests/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeArrayTests.java:58: error: primitive classes are not supported
22:06:02      [javac] 	static primitive class PointPV implements SomeIface {
```

**Comment by hangshao0** [comment]
Jenkins test sanity,extended xlinuxval jdknext

File: test/functional/Valhalla/build.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
 			<src path="${src}"/>
 			<src path="${transformerListener}" />
 			<compilerarg line='--add-exports java.base/jdk.internal.misc=ALL-UNNAMED --add-exports java.base/jdk.internal.value=ALL-UNNAMED' />
-			<compilerarg line='-source 20 -XDenablePrimitiveClasses' />
+			<compilerarg line='-source 21 -XDenablePrimitiveClasses' />
```
Comment (hangshao0) [inline test/functional/Valhalla/build.xml:59]: Jdknext is Java 22 now. We should try using 22 here.

Comment (a7ehuo) [reply test/functional/Valhalla/build.xml]: Fixed in c2a259d

**Comment by hangshao0** [comment]
Jenkins test sanity,extended xlinuxval jdknext

**Review by hangshao0 — APPROVED** [review]
Thanks for fixing this.

**Pull request merged**